### PR TITLE
Update faq.yml

### PR DESCRIPTION
**Pull request by dagdome**
new faq regioni

## Description

As already mentioned in #ISSUE_NUMBER, this PR tackles:
* ...
* ...
* ...

In particular, the ...

## Checklist

- [ ] I followed the indications in the [CONTRIBUTING](CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [ ] Ready for review! :rocket:

## Fixes

- Fixes #

### CODE REVIEW

**Comment by dagdome** [comment]
please @bfabio @denismarini

File: contents/faq-page/faq.yml [hunk 1]
```diff
@@ -116,7 +116,7 @@ questions:
       - title: È possibile accedere come nuovo rappresentante legale per una PA in cui è già presente un utente dello stesso ruolo? Cosa bisogna fare per cambiare l'utenza del rappresentante legale associata a una PA?
         content: Nel momento in cui un nuovo rappresentante legale accede alla piattaforma per conto di una PA che ha già un utente attivo in quel ruolo, il sistema invia una PEC all’amministrazione per confermare la nuova utenza e, contestualmente, disabilitare la precedente.
       - title: Senza il servizio SPID/CIE, è comunque possibile accedere a PA digitale 2026?
-        content: No. Il possesso del servizio di identità digitale (SPID o CIE) è indispensabile per accedere a PA digitale 2026.
+        content: No. Il possesso del serviziQual è l'importo riconosciuto per le attività previste dall'Avviso riferito alla Misura 1.3.1 - Regioni e Province Autonome?o di identità digitale (SPID o CIE) è indispensabile per accedere a PA digitale 2026.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:119]: Qualcosa che non va qui

File: contents/faq-page/faq.yml [hunk 2]
```diff
@@ -750,7 +750,72 @@ questions:
...
+        content: |
+          In merito al cambio di obiettivo (pacchetto) una volta scelto il pacchetto non è modificabile.  Per le variazioni al progetto fa fede l’art 14. dell’avviso, in particolare il comma 3:  "Le variazioni proposte dal Soggetto Attuatore potranno riguardare solo il cronoprogramma dell’intervento, fatte salve le scadenze previste per il conseguimento dei milestone e target associati all’investimento."
+        tag:
+          - regioni 
+      - title: In caso di due enti co-titolari (es la Regione ed una Azienda sanitaria), se entrambi espongono API, quale ente è il beneficiario del finanziamento?
+        content: |
+          Il finanziamento è riconosciuto per l’esposizione del servizio e non per la proprietà dello stesso. Nel caso di enti co-titolari si auspica un coordinamento degli stessi, anche al fine di evitare il rischio di doppio finanziamento.
+        tag:
+          - regioni 
+      - title: E’ possibile, in ambiente di Test, disporre di una API funzionante (PDND compliant) per poter effettuare prove tecniche di interoperabilità?
```
Comment (bfabio) [inline contents/faq-page/faq.yml:764]:
```suggestion
      - title: È possibile, in ambiente di Test, disporre di una API funzionante (PDND compliant) per poter effettuare prove tecniche di interoperabilità?
```

File: contents/faq-page/faq.yml [hunk 3]
```diff
@@ -750,7 +750,72 @@ questions:
...
+        tag:
+          - regioni 
+      - title: È prevista la redazione di particolare documentazione tecnica a corredo per ogni API?
+        content: |
+          Come da Modello Interoperabilità, è necessario produrre la seguente documentazione: <br> - documentazione yaml/json OpenAPI3 per REST oppure WSDL per SOAP. <br> - riferimenti alla semantica utilizzata (come da LLG Interoperabilità Tecnica §4.3).Per API di tipo REST OpenAPI3 il Dipartimento mette a disposizione un tool di verifica sintattica a questo link: https://italia.github.io/api-oas-checker/  E’ consigliabile inserire un documento allegato con indicazioni o configurazioni aggiuntive o ulteriori layer di sicurezza necessari allo scambio extra-PDND.
```
Comment (bfabio) [inline contents/faq-page/faq.yml:771]: Sarebbe meglio avere il link cliccabile qui

Comment (dagdome) [reply contents/faq-page/faq.yml:771]: Si grazie potresti inserirlo tu gentilmente ? grazie

**Pull request merged**